### ISSUE TITLE
uesr interaction disabled

### ISSUE THREAD

**Issue by Steven4294** [jiulongw/swift-unity#73]
calling     
```
currentUnityController = UnityAppController()
currentUnityController.application(application, didFinishLaunchingWithOptions: launchOptions)
```
seems to disable all user input. Not even adding the UnityView or anything, just telling the `UnityAppController` that the application has launched prevents touches from being sent.

**Comment by szczebel1995** [comment]
It probably has to do with unity initializing on the main thread. From my tests touches are received but the callbacks are fired 5-6 seconds later when main thread is free and unity finished init, I doubt that it has to do with the project setup

**Comment by VladKorzun** [comment]
Any success in fixing this issue?

**Comment by setoelkahfi** [comment]
@VladKorzun yes, I manage to solved it. Checkout [my answer](https://stackoverflow.com/a/50595308/1137814) on [this SO thread](https://stackoverflow.com/q/50403987/1137814).

If you still have problem, feel free to reply.

**Comment by VladKorzun** [comment]
@setoelkahfi thanks for a quick reply. Unfortunately, it still creates a window in the foreground that takes all interactions. Managed to solve it by removing all window creation logic from Unity generated code and replace the window with actual app window.

**Comment by setoelkahfi** [comment]
Glad that you solved it @VladKorzun . If you don't mind, please share your solution as an answer to my SO thread.

**Comment by VladKorzun** [comment]
@setoelkahfi done!

**Comment by WyattMufson** [comment]
Neither of these solved it for me

**Comment by VladKorzun** [comment]
@WyattMufson do you have a UIWindow instance in front of your UI when you inspect view hierarchy?
